.changeset/quiet-apps-find.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/mcp': patch
+---
+
+Expose MCP tool result content and metadata to UI consumers when structured output is returned.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)